### ISSUE TITLE
Add back ratings and author under Theme titles

### ISSUE THREAD

**Issue by atsay** [mozilla/addons-server#2275]
There used to be info underneath theme titles. I think they were star ratings, number of ratings, author, and #users (can't remember exactly). They have disappeared from the style refresh.

You can still see them on extensions: https://www.dropbox.com/s/0jcfln1hxszbjkw/Screenshot%202016-04-08%2010.00.19.png?dl=0

And here is what they look like for themes: https://www.dropbox.com/s/1uq7xtmhvmse5ho/Screenshot%202016-04-08%2010.00.25.png?dl=0

Please add them back, thanks.

**Comment by andymckay** [comment]
Assigning to pwalm because i think that was by design.

**Comment by tofumatt** [comment]
I believe dropping these was intentional on the part of UX, so we'll want their input here. :smile:

**Comment by pwalm** [comment]
Yep, it was intentional. We wanted to reduce the visual clutter on the page. Installing a theme is a bit different than installing an add-on: the star rating isn't a driving force for adding it to your browser.

**Comment by atsay** [comment]
Is that based on a hunch? Curious because the ability to see author and number of users helps editors feature new designers. This gives new authors a quick boost and increases engagement.

Some designs display in the browser better than others, so number of users and rating can also help people make a decision on what to install.

Lastly, authors take pride in their usage and ratings, and taking them off of all category listing pages drastically reduces their visibility.

If clutter is the concern, and star rating really has little effect on installation, perhaps add back just the number of users and author?

**Comment by atsay** [comment]
I understand the rationale for a cleaner look for users, but we shouldn't forget the designers who create the content for them.

**Comment by pwalm** [comment]
Yeah, it is based on a hypothesis, I'll cop to that. The intent is to put more focus on the theme visuals, and remove distractions and clutter.  I am open to adding back the author name (it is important to recognize the contributors in an upfront way), but I'd prefer we keep the star rating and number of users displayed on the theme detail page only for the time being.

**Comment by pwalm** [comment]
We're going to try adding back user # as well. Will have a design ready for Monday.

**Comment by pwalm** [comment]
Ok, let's try something like this:
![img_6891](https://cloud.githubusercontent.com/assets/6797635/14438164/7f698e36-ffd9-11e5-8e82-6c0ae7e0cf80.JPG)

Using this text styling for the copy we're adding back:
![screen shot 2016-04-11 at 11 36 25 am](https://cloud.githubusercontent.com/assets/6797635/14438205/bbc2334c-ffd9-11e5-9b64-4a9963131371.png)

**Comment by atsay** [comment]
Looks awesome! Thanks Phillip!

**Comment by tofumatt** [comment]
Is there any update on this? If not I think we should close it.

**Comment by atsay** [comment]
Please don't close it. @pwalm -- can you provide the design?

**Comment by pwalm** [comment]
Is that not enough guidance above? I can give actual specs later this week, got roped into some Townhall-related design work.

**Comment by tofumatt** [comment]
No rush; we were doing a style refresh triage yesterday and I didn't know if this was all that was needed or not as it was style assigned to you. I figured if nothing moved on it it wasn't too important, but seems like it is 😄 

Specs would be extra awesome, thanks 👍

**Comment by pwalm** [comment]
So let's use the same typeface and size (can't find what it is) for the user name and amount of users, but set it in regular weight and use #000000 for the colour. And not underlined/a link, obvs.

**Comment by atsay** [comment]
Can we prioritize this? Beyond community reasons, we should make it very clear on AMO that content is created by third-party developers and designers.

**Comment by tofumatt** [comment]
Prioritised! I'll aim to get this out for the next tag.

**Comment by tofumatt** [comment]
Also I would just like to apologise for being so slow on this and quick to close it. This is clearly an important issue and reading back on my comments I should have seen that! Fresh eyes make me realise recognising authorship is important.

Sorry! 😓

**Comment by ValentinaPC** [comment]
After adding back the author and ratings, there are some layout issue at hover in:
1. Themes homepage - Featured section - last row
![theme](https://cloud.githubusercontent.com/assets/15685960/17430160/55532b48-5afb-11e6-9b8a-bbb69de22a1d.gif)
2. Themes details page - More section - last row
![more](https://cloud.githubusercontent.com/assets/15685960/17430174/6d3940da-5afb-11e6-9840-a97e91148a81.png)

Verified on AMO-dev FF47(Win 7 x64).
Please let me know if this requires to be logged separately. Thanks!

**Comment by ValentinaPC** [comment]
Another scenario: if the user is signed out the new changes related to themes are not listed in some places (themes homepage, any theme category)
Please see the screenshot: 
![2016-08-05_1635](https://cloud.githubusercontent.com/assets/15685960/17438461/a146ed02-5b2b-11e6-85c6-83a535e3df35.png)

**Comment by tofumatt** [comment]
That seems to be a caching issue, as the HTML was updated. We'll need to force a refresh of the cache or something.

@jasonthomas Is the cache reset on dev/stage/prod after a deploy?

**Comment by ValentinaPC** [comment]
Verified as fixed for "Featured Themes" section and "More" section in AMO-dev using FF47 (Win 7 x64) (logged in or out).
Postfix videos: 
![featured th](https://cloud.githubusercontent.com/assets/15685960/17471222/5361c86a-5d4b-11e6-9579-a7a5f8e16d80.gif)
![more](https://cloud.githubusercontent.com/assets/15685960/17471226/5b68a2a4-5d4b-11e6-9d96-423865efc789.gif)

Still reproducing for all themes categories in Up & Coming section, last row. (logged in or out)
Please the video: 
![theme cat](https://cloud.githubusercontent.com/assets/15685960/17471256/98eea6fa-5d4b-11e6-8ab0-e70dd9bd0fc0.gif)

**Comment by ValentinaPC** [comment]
**3.** There is also a small display issue in Collections that only contain themes.
Please see the video: 
![collections](https://cloud.githubusercontent.com/assets/15685960/17472144/f0eeafe8-5d51-11e6-88f3-a34d7ac5393d.gif)

**Comment by tofumatt** [comment]
Sorry, can you give URLs to the issues? I can't reproduce.

**Comment by ValentinaPC** [comment]
-> Themes categories (hover over second row): https://addons-dev.allizom.org/en-US/firefox/themes/music
-> Collections (containing only themes): https://addons-dev.allizom.org/en-US/firefox/collections/TesterW/only-themes/

**Comment by atsay** [comment]
Thanks Tofu & Valentina, really appreciate you getting this out.
